concepts/0006-ssi-notation/README.md

```diff
@@ -51,334 +33,151 @@ This notation aims to be:
 
-Two types of entities are being defined. Things and Identity Owners.
-Each have their own sections below and define their own syntax.
+Instead, we want something simple, clean, and universally supported. Hence the 7-bit ASCII requirement. A future version of this RFC, or an addendum to it, might explain how to map this 7-bit ASCII notation to various schemes that use mathematical symbols and are familiar to experts from other fields.
 
-![taxonomy](taxonomy.png)
+### Solution
 
-### Identity Owners
+#### Controllers and Subjects
 
-In a self-sovereign worldview, the conceptual center of gravity is
-__identity owners__. These are people and institutions--the type of
-entity that can (at least theoretically) be held legally accountable
-for its actions. Identity owners control a __sovereign domain__ that
-encompasses all their agents, data, and devices. The notation's first
-goal is therefore an efficient and unambiguous way to anchor
-derivative concepts to owners and their domains.
+An identified thing (the referent of an identifier) is called an __identity subject__. Identity subjects can include:
+ 
+ * passive things (pets, digital documents, symphonies, rivers, asteroids, strains of bacteria...)
+ * active things (people, institutions, IoT devices)
+ 
+The latter category may also act as an __identity controller__ -- something that projects its intent with respect to identity onto the digital landscape.
 
-Identity owners are denoted with a single
-capital ASCII alpha, often corresponding to their first initial. For
-example, Alice might be represented as `A`. By preference, the first
-half of the alphabet is used (because "x", "y", and "z" tend to have
-other ad-hoc meanings). When reading aloud, the spoken form of
-a symbol like this is the name of the letter. The relevant [ABNF](
-https://tools.ietf.org/html/rfc5234) fragment is:
+When an identity controller controls its own identity, we say that it has __self sovereignty__ -- and we call it a __self__. (The term __identity owner__ was originally used for an identity controller managing itself, but this hid some of the nuance and introduced [legal concepts of ownership that are problematic](https://medium.com/@hackylawyER/do-we-really-want-to-sell-ourselves-the-risks-of-a-property-law-paradigm-for-data-ownership-b217e42edffa), so we'll avoid it here.)
+ 
+In our notation, selves are denoted with a single *upper-case* ASCII alpha, often corresponding to a first initial of their human-friendly name. For example, Alice might be represented as `A`. By preference, the first half of the alphabet is used (because "x", "y", and "z" tend to have other ad-hoc meanings). When reading aloud, the spoken form of a symbol like this is the name of the letter. The relevant [ABNF]( https://tools.ietf.org/html/rfc5234) fragment is:
```

Comment:
With the change from identity owner -> identity controller, should Identity controller be referenced here?

>In our notation, selves (or identity controllers) are denoted...

It's referenced in the section above, maybe an addition here would solidify the point?
or am I missing a nuance? I read selves and think people/institutions etc

Reply:
Tweaked per @moosey's suggestion.